pokemongo_bot/cell_workers/follow_path.py

```diff
@@ -77,10 +77,12 @@ def load_json(self):
                     'position': point_tuple
                 }
             )
-            # Keep point['location']
+            # Keep point['location'] and other keys
             point["lat"] = float(point_tuple[0])
             point["lng"] = float(point_tuple[1])
             point["alt"] = float(point_tuple[2])
+            if "mode" not in point:
+                point["mode"] = "walking"
```

Comment:
Replace this if block for `point["mode"] = mode.get("mode", "walking")`. If a mode has been set, it'll stay as it is, otherwise assign "walking" to it.